aggregators/will-get-add-to-homescreen-prompt/index.js

```diff
@@ -18,23 +18,67 @@
 'use strict';
 
 const Aggregate = require('../aggregate');
-const AddToHomescreen = require('./add-to-homescreen');
-const WorksOffline = require('./works-offline');
-const MobileFriendly = require('./mobile-friendly');
 
-class WillGetAddToHomescreenPrompt extends Aggregate {
+class AddToHomescreen extends Aggregate {
 
   static get name() {
     return 'Will Get Add to Homescreen Prompt';
   }
 
+  /**
+   * For the install-to-homescreen / install-app-banner prompt to show,
+   * Chrome needs the following:
+   *   - has a registered SW
+   *   - has valid manifest
+   *   - valid start_url
+   *   - valid name
+   *   - valid short_name
+   *   - icon of size >= 144x144 and png (either type `image/png` or filename ending in `.png`
+   * More details: https://github.com/GoogleChrome/lighthouse/issues/23
+   *
+   * TODO: We should allow icons >=144, rather than >= 192
```

Comment:
we shouldn't close #23 until this is fixed. The change shouldn't be long in the icons-192 audit. Split on `'x'` (which is required to be lowercase after parsing), `parseInt` on each, then check that both width and height are >= 144